configs/mmdet/_base_/base_instance-seg_static_solo.py

```diff
@@ -0,0 +1,4 @@
+_base_ = ['./base_static.py']
+
+onnx_config = dict(output_names=['dets', 'labels', 'masks'])
+codebase_config = dict(post_processing=dict(export_postprocess_mask=True))
```

Comment:
Why do we have to export postprocess mask for solo only?

Reply:
> Why do we have to export postprocess mask for solo only?

SOLO calculate the mask directly, while other models generate masks from det bboxes, which is dummy in SOLO.